app/lib/analyze_function.rb

```diff
@@ -2,9 +2,19 @@
 # Created at: 30.11. 2014
 #
 # Description: Vytvorenie a naplnenie generickej tabulky na zaklade csv suboru.
+require 'cmd_interface'
 
 class AnalyzeFunction
 
+def r_analyze_dataset(dataset)
+  Thread.new do
+    dataset_id = dataset.id
+    path = dataset.storage
+    cmd = "Rscript app/lib/r/analyze.R #{path} #{dataset_id}"
+    CMDInterface.new.Exec_command(cmd)
```

Comment:
resp. mozem doplnit do exec_command ze raise error..ked bude error a ty to das do begin rescure ...